scripts/pre_commit_linter.py

```diff
@@ -2379,10 +2265,11 @@ def _check_directive_scope(self):
                 summary_messages.append(summary_message)
 
             python_utils.PRINT('')
-            self.process_manager['directive'] = summary_messages
-            _STDOUT_LIST.append(stdout)
+            process_manager['directive'] = summary_messages
+            global_stdout.append(stdout)
 
-    def _check_sorted_dependencies(self):
+    def _check_sorted_dependencies(
+            self, all_filepaths, global_stdout, process_manager):
```

Comment:
Can we add docstring for all the params here and elsewhere?

Reply:
Will do, thanks!